episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+We’ll use the pipe `%>%` to connect some of their functions, so let’s also call to the `{tidyverse}` package:
+
+```{r,message=FALSE,warning=FALSE}
+library(cfr)
+library(epiparameter)
+library(tidyverse)
+```
+
+## Naive CFR
+
+We measure disease severity in terms of case fatality ratio (CFR) --which is given by the ratio of the cumulative number of deaths $D_{t}$ to the cumulative number of confirmed cases $C_{t}$ at a certain time $t$. We can refer to the _naive CFR_ (also crude or biased CFR, $b_{t}$):
+
+$$ b_{t} =  \frac{D_{t}}{C_{t}} $$
+
+<!-- total number of deaths $D$ divided by total number of confirmed cases $C$ -->
+
+:::::::::::::: callout
+
+### What are data sources to estimate CFR?
+
+[Verity et al., 2020](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30243-7/fulltext) summaries the spectrum of COVID-19 cases.
+
+![Spectrum of COVID-19 cases. The CFR aims to estimate the proportion of Deaths among confirmed cases in an epidemic.](fig/cfr-spectrum-cases-covid19.jpg)
+
+- At the top of the pyramid, those meeting the WHO case criteria for **severe** or critical cases will likely be identified in the hospital setting, presenting with atypical viral pneumonia. These cases will have been identified in mainland China and among those categorised internationally as local transmission. 
+- Many more cases are likely to be **symptomatic** (i.e., with fever, cough, or myalgia) but might not require hospitalisation. These cases will have been identified through links to international travel to high-risk areas and through contact-tracing of contacts of confirmed cases. They might also be identified through population surveillance of, for example, influenza-like illness. 
+- The bottom part of the pyramid represents **mild** (and possibly **asymptomatic**) cases. These cases might be identified through contact tracing and subsequently via serological testing.
+
+::::::::::::::
+
+To calculate the naive CFR, the `{cfr}` package requires a data frame with three columns named:
+
+- `date`
+- `cases`
+- `deaths`
+
+:::::::::::::::::: callout
+
+### We need daily data
+
+These should give the **daily** number of reported cases and deaths. 
+
+We emphasise **daily** because this needs to show all the observations for all the days in the time frame of your analysis, even if the number of observations is zero or there are missing values.
+
+Also, this currently only works for **daily** data, not weekly data.
+
+<!-- suggest ways to deal with raw input weekly data -->
+<!-- https://github.com/epiverse-trace/cfr/issues/117 -->
+
+::::::::::::::::::
+
+Let's explore the `ebola1976` dataset, which comes from the first Ebola outbreak in Zaire in 1976, as analysed by Camacho et al. (2014).
```

Comment:
Zaire was an interim name for DRC under military rule - which is when this outbreak occurred - but it's good to use the modern (and original) name.

Reply:
edited in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/072a64b3f3e4b9ce982a0a69ce9072b95f54d792